resources/views/wiki/show.blade.php

```diff
@@ -28,12 +28,16 @@
             <div class="osu-page-header__title-box">
                 @if (present($page->subtitle()))
                     <h2 class="osu-page-header__title osu-page-header__title--small">
-                        {{ $page->subtitle() }}
+                        <a class="osu-page-header__link" href="{{ wiki_url($page->parentPath(), $page->requestedLocale) }}">
+                            {{ $page->subtitle() }}
+                        </a>
                     </h2>
                 @endif
 
                 <h1 class="osu-page-header__title osu-page-header__title--main">
-                    {{ $page->title() }}
+                    <a class="osu-page-header__link" href="{{ wiki_url($page->path, $page->requestedLocale) }}">
```

Comment:
does the page need to link back to itself in the title?

Reply:
idk, I just added it to be consistent with the other title. it provides an easy rightclick->copy for sharing the page too

Reply:
Title either shouldn't be linked or have the hover effect removed (basically, shouldn't look like a link).